DataFormats/Math/interface/alpaka/deltaPhi.h

```diff
@@ -0,0 +1,35 @@
+#ifndef DataFormats_Math_alpaka_deltaPhi_h
+#define DataFormats_Math_alpaka_deltaPhi_h
+
+#include "HeterogeneousCore/AlpakaInterface/interface/config.h"
```

Comment:
```suggestion
#include <alpaka/alpaka.hpp>
```
would seem to be sufficient.